docs/release-notes/self-hosted/self-hosted-v11.0.0.md

```diff
@@ -0,0 +1,94 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Self-hosted v11.0.0.
+codacy_tools_version_old: https://github.com/codacy/codacy-tools-sh/releases/tag/sh-1.1.3
+codacy_tools_version_new: https://github.com/codacy/codacy-tools-sh/releases/tag/sh-1.1.6
+---
+
+# Self-hosted v11.0.0
+
+These release notes are for [Codacy Self-hosted v11.0.0](https://github.com/codacy/chart/releases/tag/11.0.0){: target="_blank"}, released on April 19, 2023.<!-- TODO Update release date -->
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Upgrading Codacy Self-hosted
+
+Follow the steps below to upgrade to Codacy Self-hosted v11.0.0:
+
+1.  Check the [release notes for all Codacy Self-hosted versions](../index.md#self-hosted) **between your current version and the most recent version** for breaking changes and follow the instructions provided <span class="skip-vale">carefully</span>.
+
+1.  Follow the instructions to [upgrade your Codacy Self-hosted instance](https://docs.codacy.com/v11.0/chart/maintenance/upgrade/).
+
+1.  Update your Codacy command-line tools to the following versions:<!--TODO Check CLI tool versions-->
+
+    -   [Codacy Analysis CLI 7.7.4](https://github.com/codacy/codacy-analysis-cli/releases/tag/7.7.4)
+    -   [Codacy Coverage Reporter 13.12.3](https://github.com/codacy/codacy-coverage-reporter/releases/tag/13.12.3)
```

Comment:
@MiguelMarcelino, what are the latest command-line tool versions that are compatible with Codacy Self-hosted 11.0.0? These were copied from version 10.0.0.

Reply:
Hello Paulo! I noticed that the latest release for the CLI failed the workflow in the CI: https://app.circleci.com/pipelines/github/codacy/codacy-analysis-cli/1493/workflows/9dc59457-63aa-4299-904d-507dc5fc3b10
I think we will have to go with the latest successfully released version, which is still version `7.7.4`.

Reply:
I have some updates on this:
* Codacy Analysis CLI is now on version `7.8.3`.
* Codacy Coverage Reporter is now on version `13.13.0`.

Reply:
Thanks @MiguelMarcelino, I've just updated the pull request with the correct versions. :+1: